modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/CSDiagramEntryInfoItemFields.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.content.web.internal.info;
+
+import com.liferay.commerce.shop.by.diagram.model.CSDiagramEntry;
+import com.liferay.info.field.InfoField;
+import com.liferay.info.field.type.BooleanInfoFieldType;
+import com.liferay.info.field.type.DateInfoFieldType;
+import com.liferay.info.field.type.NumberInfoFieldType;
+import com.liferay.info.field.type.TextInfoFieldType;
+import com.liferay.info.localized.InfoLocalizedValue;
+
+/**
+ * @author Mahmoud Azzam
+ */
+public class CSDiagramEntryInfoItemFields {
+
+	public static final InfoField<NumberInfoFieldType> companyIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"companyId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "company-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cpDefinitionIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cpDefinitionId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "cp-definition-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cpInstanceIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cpInstanceId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "cp-instance-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cProductIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cProductId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "c-product-id")
```

Comment:
why didn't you use cProductId as in CPDefinitionInfoItemFields.java class?

Reply:
Same here, cProductId does not exist in Language.properties

Reply:
```suggestion
				CSDiagramEntryInfoItemFields.class, "cProductId")
```